plugins/outputs/whatap/whatap.go

```diff
@@ -0,0 +1,382 @@
+package whatap
+
+import (
+	"fmt"
+	"log"
+	"net"
+	"os"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/influxdata/telegraf"
+
+	//"github.com/influxdata/telegraf/internal"
+	"github.com/influxdata/telegraf/plugins/outputs"
+)
+
+const (
+	NETSRC_AGENT_ONEWAY = 10
+	VALUE_DECIMAL       = 20
+	VALUE_DOUBLE        = 40
+	VALUE_TEXT          = 50
+	VALUE_MAP           = 80
+	// 5633
+	PACK_TAG_COUNT = 0x1601
+)
+
+type Whatap struct {
+	License string        `toml:"license"`
+	Servers []string      `toml:"servers"`
+	Pcode   int64         `toml:"pcode"`
+	Timeout time.Duration `toml:"timeout"`
```

Comment:
Please use
```suggestion
	Timeout config.Duration `toml:"timeout"`
```
instead to allow suffices like `ms`, `s`, `m` or `h`.

Reply:
Edited and applied. We ask for confirmation.